cfgov/paying_for_college/views.py

```diff
@@ -309,6 +311,27 @@ def school_search_api(request):
     return HttpResponse(json_doc, content_type='application/json')
 
 
+def school_search(request):
+    # If there's no query string, don't search
+    search_term = request.GET.get('q', '').strip()
+    if not search_term:
+        return HttpResponse(json.dumps([]), content_type='application/json')
+    response = SchoolSearch(search_term).search()
+
+    document = [{'schoolname': school.text,
+                 'id': school.school_id,
+                 'city': school.city,
+                 'nicknames': school.nicknames,
+                 'state': school.state,
+                 'zip5': school.zip5,
+                 'url': reverse("paying_for_college:disclosures:school-json",
```

Comment:
Depending on time, it may be easier to just index the url field and map it here instead of calling reverse on each result everytime it gets returned.

Reply:
I will look into the changes needed to add the `url` field into the `SchoolDocument` class.

Reply:
Also removed `reverse` from `url` in `school_search`